applications/nrf5340_audio/src/audio/sw_codec_select.c

```diff
@@ -20,20 +21,82 @@ LOG_MODULE_REGISTER(sw_codec_select, CONFIG_SW_CODEC_SELECT_LOG_LEVEL);
 
 static struct sw_codec_config m_config;
 
+static struct sample_rate_converter_ctx encoder_converters[AUDIO_CH_NUM];
+static struct sample_rate_converter_ctx decoder_converters[AUDIO_CH_NUM];
+
+/**
+ * @brief	Converts the sample rate of the uncompressed audio stream if needed.
+ *
+ * @details	Two buffers must be made available for the function: the input_data buffer which
+ *		contain the samples for the audio stream, and the conversion buffer that will be
+ *		used to store the converted audio stream. data_ptr will point to conversion_buffer
+ *		if a conversion took place, otherwise it will point to input_data.
```

Comment:
```suggestion
 * @details	Two buffers must be made available for the function: the input_data buffer that
 *		contains the samples for the audio stream, and the conversion buffer that will be
 *		used to store the converted audio stream. data_ptr will point to conversion_buffer
 *		if a conversion took place; otherwise, it will point to input_data.
```